app/src/server/api/routers/panel.ts

```diff
@@ -0,0 +1,63 @@
+import { createTRPCRouter, protectedProcedure } from "@/server/api/trpc";
+import { baseServerResponse, errorResponse } from "@/server/api/trpc";
+import { eq } from "drizzle-orm";
+import { reportLog, userData } from "@/drizzle/schema";
+import { fetchUser } from "@/routers/profile";
+import { getServerPusher, updateUserOnMap } from "@/libs/pusher";
+import type { UserStatus } from "@/drizzle/constants";
+import { z } from "zod";
+import { inferRouterOutputs } from "@trpc/server";
+import { nanoid } from "nanoid";
+
+export const panelRouter = createTRPCRouter({
```

Comment:
Maybe we should name this one staffRouter for clarify?

Reply:
Renamed